src/test/java/org/opentripplanner/updater/trip/TripUpdateBuilder.java

```diff
@@ -88,6 +88,21 @@ public TripUpdateBuilder addStopTime(
     return addStopTime(null, -1, stopSequence, NO_VALUE, NO_VALUE, scheduleRelationship, null);
   }
 
+  /**
+   * Add a skipped stop to the TripUpdate.
+   */
+  public TripUpdateBuilder addSkippedStop(int stopSequence) {
```

Comment:
How about I add another method for NO_DATA cases which also don't have times associated, and remove the `addStopTime` method I mentioned earlier, so that the StopTimes can be built more explicitly?

Reply:
Yes, that's a good idea.